homeassistant/components/analytics/analytics.py

```diff
@@ -222,6 +228,12 @@ async def send_analytics(self, _=None) -> None:
             if supervisor_info is not None:
                 payload[ATTR_ADDONS] = addons
 
+            if ENERGY_DOMAIN in integrations:
+                manager = await async_get_energy_manager(self.hass)
+                payload[ATTR_ENERGY] = {
+                    ATTR_CONFIGURED: manager.data != manager.default_preferences()
```

Comment:
So far the manager has just been used by the energy integration. Maybe we should not expose it directly to other integrations but instead create dedicated functions that return the configured energy integration state etc?

Reply:
For now, we only want if it's configured or not, but I assume that will expand in the future.
I wonder if it makes sense to add a system health function to the energy integration, and then the analytics integration can use that.